ca_driver/src/create_driver.cpp

```diff
@@ -568,10 +578,33 @@ void CreateDriver::publishBumperInfo()
   bumper_pub_.publish(bumper_msg_);
 }
 
+void CreateDriver::publishCliffInfo()
+{
+  cliff_msg_.header.stamp = ros::Time::now();
+
+  if (model_.getVersion() >= create::V_3)
```

Comment:
I have the hardware to test `V_2` and `V_3` functionality and will do testing before merging this PR.